src/content/docs/en/install/auto.mdx

```diff
@@ -1,154 +1,420 @@
+    - package.json
+    - tsconfig.json
+    </FileTree>
+
+8. You can now [start the Astro dev server](#start-the-astro-dev-server) and see a live preview of your project while you build!
+
+</Steps>
+
+
+## Start the Astro dev server
+
+    Astro comes with a built-in development server that has everything you need for project development. The `astro dev` CLI command will start the local development server so that you can see your new website in action for the very first time.
+
+    Every starter template comes with a pre-configured script that will run `astro dev` for you. After navigating into your project directory, use your favorite package manager to run this command and start the Astro development server.
+
+    <Tabs>
+      <TabItem label="npm">
+      ```shell
+      npm run dev
+      ```
+      </TabItem>
+      <TabItem label="pnpm">
+      ```shell
+      pnpm run dev
+      ```
+      </TabItem>
+      <TabItem label="yarn">
+      ```shell
+      yarn run dev
+      ```
+      </TabItem>
+    </Tabs>
+
+
+If all goes well, Astro should now be serving your project on [http://localhost:4321/](http://localhost:4321/). Visit that link in your browser and see your new site!
+
+## Work in development mode
+
+Astro will listen for live file changes in your `src/` directory and update your site preview as you build, so you will not need to restart the server as you make changes during development. You will always be able to see an up-to-date version of your site in your browser when the dev server is running.
+
+When viewing your site in the browser, you'll have access to the [Astro dev toolbar](/en/guides/dev-toolbar/). As you build, it will help you inspect your [islands](/en/concepts/islands/), spot accessibility issues, and more.
+
+If you aren't able to open your project in the browser after starting the dev server, go back to the terminal where you ran the `dev` command and check the message displayed. It should tell you if an error occurred, or if your project is being served at a different URL than [http://localhost:4321/](http://localhost:4321/).
+
+## Build and preview your site
+
+To check the version of your site that will be created at build time, quit the dev server and run the appropriate build command for your package manager in your terminal:
+
+    <Tabs>
+      <TabItem label="npm">
+      ```shell
+      npm run build
+      ```
+      </TabItem>
+      <TabItem label="pnpm">
+      ```shell
+      pnpm build
+      ```
+      </TabItem>
+      <TabItem label="yarn">
+      ```shell
+      yarn run build
+      ```
+      </TabItem>
+    </Tabs>
+
+Astro will build a deploy-ready version of your site in a separate folder (`dist/` by default) and you can watch its progress in the terminal. This will alert you to any build errors in your project before you deploy to production. If you opted to use Typescript, the `build` script will also check your project for type errors.
```

Comment:
```suggestion
Astro will build a deploy-ready version of your site in a separate folder (`dist/` by default) and you can watch its progress in the terminal. This will alert you to any build errors in your project before you deploy it to production. If you opted to use Typescript, the `build` script will also check your project for type errors.
```

Reply:
I think the phrase people are more likely to hear in English is "deploy to production" (not "deploy it to production") although both are correct. In a case like this, we will often choose the phrase we think most people will use or say, so that if someone is new, they are encountering the phrase the way they will hear other people talk about it.